client/v2/autocli/flag/binary.go

```diff
@@ -0,0 +1,63 @@
+package flag
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"github.com/cockroachdb/errors"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"os"
+)
+
+type binaryType struct{}
+
+var _ Value = (*fileBinaryValue)(nil)
+
+func (f binaryType) NewValue(_ context.Context, _ *Builder) Value {
+	return &fileBinaryValue{}
+}
+
+func (f binaryType) DefaultValue() string {
+	return ""
+}
+
+// fileBinaryValue is a Value that holds a binary file.
+type fileBinaryValue struct {
+	value []byte
+}
+
+func (f *fileBinaryValue) Get(protoreflect.Value) (protoreflect.Value, error) {
+	return protoreflect.ValueOfBytes(f.value), nil
+}
+
+func (f *fileBinaryValue) String() string {
+	return string(f.value)
+}
+
+// Set implements the flag.Value interface for binary files, with exceptions.
+// If the input string is a valid file path, the value will be the content of that file.
+// If the input string is a valid hex or base64 string, the value will be the decoded form of that string.
+// If the input string is not a valid file path, hex string, or base64 string, Set will return an error.
+func (f *fileBinaryValue) Set(s string) error {
+	if data, err := os.ReadFile(s); err == nil {
```

Comment:
Why don't we start with the cheapest code first that tries:
1. Hex decoding
2. Base64 decoding
3. File reading 


Otherwise firstly trying file reading is going to incur expenses especially if the main use case is decoding. Or is it required to firstly attempt the file?

Reply:
I like it, gonna make other PR

Reply:
I think the file should stay first. So if you have a file called `61` it will read it instead of wrongly reading this as a hex value.

Reply:
This is a good point!

Reply:
File first